www/docs/configuration/callbacks.md

```diff
@@ -146,18 +123,14 @@ callbacks: {
 ```
 
 :::tip
-Use an if branch in jwt with checking for existence of any other params than token. If any of those exist, you call jwt for the first time.
-This is a good place to add for example an `access_token` to your jwt, if you want to.
-:::
-
-:::tip
-Check out the content of all the params in addition `token`, to see what info you have available on signin.
+Use an if branch in jwt with checking for existence of any other params than `token`. If any of those exist, you call jwt for the first time.
```

Comment:
Bit confusingly worded imo, how about something like, "Use an if branch in the jwt callback to check for the existence of any other params in addition to `token`. If any exist, you can call jwt for the first time." ?

Reply:
Thanks, I now reworded this section!